clang/test/SemaSYCL/constexpr-function-pointer.cpp

```diff
@@ -0,0 +1,46 @@
+// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify -sycl-std=2020 -std=c++17 %s
+
```

Comment:
Could you please add a comment what test is doing?

Reply:
Sure, done.

Reply:
Thank you.